eiffel-syntax-and-usage/event-schemas.md

```diff
@@ -23,26 +23,26 @@ The documentation of each event is available in a Markdown file that covers the
-| `_version`      | The version of the event or other type, e.g. "3.1.0". This string must match the name of the definition file, except for the filename suffix. |
-| `_abbrev`       | The abbreviation of the event name, e.g. "CD" for EiffelCompositionDefinedEvent. |
-| `_description`  | An overall description of the event. |
-| `_links`        | An object describing the valid link types for the event. |
-| `_links.<link type>.description`  | A description of the link type. |
-| `_links.<link type>.required`     | A boolean value indicating whether a link of this type is required. Optional, default false. |
-| `_links.<link type>.multiple`     | A boolean value indicating whether multiple links of this type is allowed. Optional, default false. |
-| `_links.<link type>.experimental` | A boolean value indicating whether the link type is experimental, i.e. the only valid target is an experimental event type. Optional, default false. |
-| `_links.<link type>.targets.any_type`  | A boolean value indicating whether the link can point to an event of any type. |
-| `_links.<link type>.targets.types`     | A string array of event names that the link type may point to. Must be non-empty if `any_type` is false, and must be empty if `any_type` is true. |
+| Key             | Description                                                                                                                                            |
+| --------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `_name`         | The name of the type, e.g. "EiffelCompositionDefinedEvent". This string must match the name of the parent directory.                                   |
+| `_version`      | The version of the event or other type, e.g. "3.1.0". This string must match the name of the definition file, except for the filename suffix.          |
+| `_abbrev`       | The abbreviation of the event name, e.g. "CD" for EiffelCompositionDefinedEvent.                                                                       |
+| `_description`  | An overall description of the event.                                                                                                                   |
+| `_links`        | An object describing the valid link types for the event.                                                                                               |
+| `_links.<link type>.description`  | A description of the link type.                                                                                                                        |
+| `_links.<link type>.required`     | A boolean value indicating whether a link of this type is required. Default false.                                                                     |
```

Comment:
What does a default value mean for a required property?

Reply:
Remove default

Reply:
Fixed in [c47ec58](https://github.com/eiffel-community/eiffel/pull/402/commits/c47ec589ce74fbe97c471bddba5b335b73507f52)